app/src/main/java/eu/kanade/tachiyomi/util/system/ContextExtensions.kt

```diff
@@ -162,8 +163,8 @@ fun Context.hasPermission(permission: String) = ContextCompat.checkSelfPermissio
     }
 }
 
-val getDisplayHeightInPx: Int
-    get() = Resources.getSystem().displayMetrics.heightPixels
+val getDisplayMaxHeightInPx: Int
```

Comment:
No need to rename

Reply:
You know what rename it to this
```suggestion
val displayBiggerSideSizeInPx: Int
```

Reply:
I think max in the name shows that the value is not always the same reference so it will be clearer.